### ISSUE TITLE
Maximum call stack size exceeded when running `ng xi18n` after updating to Angular 9

### ISSUE THREAD

**Issue by ignlg** [angular/angular-cli#17133]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [x] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

 Yes, the previous version in which this bug was not present was: 8.2.14

### Description

After updating to Angular 9.0.4, `ng xi18n` crashes.

Contrary to other potentially related issues, `ng build` and `ng serve` work fine and the application works fine (excluding i18n support).

I have tried to remove and rebuild `node_modules` and looked for circular imports with Madge. Nothing worked.

Tried with every Angular 9.0.x version up to 9.0.4. Same result.

## 🔬 Minimal Reproduction

Files:
```
lib/helpers/example.helper.ts
lib/helpers.ts
```
where `helpers.ts` content is
```
export * from './helpers';
```

Include it in `tsconfig.json`:
```
{
  "include": [ "lib/",  . . . ],
  . . .
}
```
Run:
```
ng xi18n
```

## 🔥 Exception or Error
<pre><code>
ERROR in Maximum call stack size exceeded
</code></pre>

As there is not a `--verbose` option for `xi18n`, this is the only output I can get.

## 🌍 Your Environment
<pre><code>

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/

Angular CLI: 9.0.4
Node: 12.16.1
OS: darwin x64

Angular: 9.0.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, localize, platform-browser
... platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.4
@angular-devkit/build-angular     0.900.4
@angular-devkit/build-optimizer   0.900.4
@angular-devkit/build-webpack     0.900.4
@angular-devkit/core              9.0.4
@angular-devkit/schematics        9.0.4
@ngtools/webpack                  9.0.4
@schematics/angular               9.0.4
@schematics/update                0.900.4
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
</code></pre>

**Anything else relevant?**

Could be related to #17106

**Comment by clydin** [comment]
This appears to be a bug but we will need to look at a reproduction to find and fix the problem. Can you setup a minimal reproduction please? 

You can read [here](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) why this is needed. A good way to make a minimal reproduction is to create a new app via `ng new repro-app` and adding the minimum possible code to show the problem. Then you can push this repository to github and link it here.

**Comment by ignlg** [comment]
You are right. As this emerged in a big project, it is not trivial to reduce it to the minimum. I will have something defined no later than the current week. Thanks

**Comment by jnoga** [comment]
I'm facing the same isuse with `ng xi18n`. 

I was able to get the result error log, It's seems to be a `webpack` error:

[xi18n.error.log](https://github.com/angular/angular-cli/files/4312350/xi18n.error.log)

Based on #17106 I installed also the `terser@lastest` version with the same result.

Any clue?

**Comment by alan-agius4** [comment]
Hi @ignlg, I am not seeing any link to a reproduction repo in your issue.

Can you please provide a link? Thanks.

**Comment by alan-agius4** [comment]
Thanks for reporting this issue. However, you didn't provide sufficient information for us to understand and reproduce the problem. Please check out [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) to understand why we can't act on issues that are lacking important information.

If the problem persists, please file a new issue and ensure you provide all of the required information when filling out the issue template.

**Comment by jnoga** [comment]
@alan-agius4  It was fixed with the 9.1.0 version, sorry for not updating this issue, thanks anyway!

**Comment by alan-agius4** [comment]
Glad to hear that the issue has been addressed 😊
